zenodo/config.py

```diff
@@ -744,7 +748,8 @@ def _(x):
 #: Default database host.
 SQLALCHEMY_DATABASE_URI = os.environ.get(
     "SQLALCHEMY_DATABASE_URI",
-    "postgresql+psycopg2://localhost/zenodo")
+    #  "postgresql+psycopg2://localhost/zenodo")
+    'sqlite:///pippo.db')
```

Comment:
If you need to modify default configuration please use `` <VIRTUAL_ENV>/instance/zenodo.cfg`` to avoid that unintended errors sneak in :-)